### ISSUE TITLE
remote players show duplicates when browsing library

### ISSUE THREAD

**Issue by parejkoj** [owntone/owntone-server#52]
I submitted a bug against the Ubuntu package about this, and emailed the Debian maintainer who suggested I report it here. I'll copy some of the Ubuntu bug report below:

https://bugs.launchpad.net/ubuntu/+source/forked-daapd/+bug/1362395

When playing a from a forked-daapd server (0.21 on Trusty built via dpkg-buildpackage), using either iTunes 11.3.1 OSX, Android DAAP Media Player, or Android Music Pump, I see multiple copies of some songs. Not all songs or albums are duplicated, only some.

I have two directories in the library:directory section of my /etc/forked-daapd.conf file; one for music, one for movies. The bug appears whether I include the movie folder or not.

I'm happy to provide any extra information (log files, config, etc.) that may be of assistance.

Your recent commit, 2545aedc2ab073980fdd985d35637f99c994af72, mentions avoiding duplicates in the admin table. Is this the same issue or a fix for something else?

**Comment by parejkoj** [comment]
Thanks for the initial response, and I'm sorry for the long delay.

I finally did a bit of digging, and it turns out the problem was that I had extra copies of some songs in a separate directory. I made a side directory for mp3 conversions to put on my phone, and it was also being scanned, resulting in the duplications.

So, the original bug is not a problem with daapd, but was my fault. Sorry for the trouble.

That said, is there a way to add an "ignore list", that skips over paths (either matching some regex, or just everything under a given oath)? That would solve this problem without requiring that I move my converted files to a different directory.
